cmake/custom/binary-info.cmake

```diff
@@ -48,6 +48,7 @@ if(GIT_FOUND)
     )
 endif()
 
+message(STATUS "MRCPP version: ${MRCPP_VERSION}")
 message(STATUS "Git branch: ${_git_branch}")
 message(STATUS "Git last commit hash: ${_git_describe}")
```

Comment:
To avoid confusion, I would change the name of the `_git_describe` variable to `_git_last_commit_hash`

Reply:
done